src/sidebar/store/modules/activity.js

```diff
@@ -24,6 +24,11 @@ function init() {
      * Have annotations ever been fetched?
      */
     hasFetchedAnnotations: false,
+    /**
+     * The number of total results (annotations) the service reported as
+     * matching the most recent load/search request
+     */
+    resultCount: null,
```

Comment:
Given that this module is not about annotations specifically `resultCount` is perhaps ambiguous here. It should be easy enough to change when the need arises though.

Reply:
Yes, naming did slow me down a little here initially. Ergo my careful commenting. I'll give this a momentary think.

Reply:
I'm renaming this attribute and the related selector/action creator:

* `annotationResultCount`
* `annotationResultCount()` selector
* `setAnnotationResultCount()` action creator